chain-impl-mockchain/src/certificate/vote_plan.rs

```diff
@@ -427,12 +443,22 @@ impl Readable for VotePlan {
             proposals.proposals.push(proposal);
         }
 
+        let member_keys_len = buf.get_u64()?;
+        let mut committee_public_keys = Vec::new();
+        for _ in 0..member_keys_len {
+            let key_len = buf.get_u64()?;
+            let key_buf = buf.get_slice(key_len as usize)?;
+            // Unwrap should be ok here, since we did serialize it ourselves
+            committee_public_keys.push(MemberPublicKey::from_bytes(key_buf).unwrap());
```

Comment:
This is a validating reader from storage format, and if the length is wrong it will blow up. I don't think it's OK.